kinetic/distribution.yaml

```diff
@@ -2154,6 +2154,17 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: kinetic-devel
     status: maintained
+  fluent_logger_ros:
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/akio/ros_fluent_logger-release.git
+      version: 0.1.0-0
+    source:
+      type: git
+      url: https://github.com/akio/fluent_logger_ros.git
+      version: 0.1.0
```

Comment:
Looks like the CI is failing because the source version needs to be a branch and not a tag

Reply:
@akio Friendly ping.

Reply:
@akio It looks like this line should be the branch `kinetic-devel` please update it in this branch.

It's been a week since you submitted this for indexing. If you don't respond in the next day or two we'll close this due to no response. You're welcome to reopen or comment back here if you've updated after that.